runtime/src/accounts_db.rs

```diff
@@ -3645,10 +3645,11 @@ impl AccountsDb {
                 // will be able to find the account in storage
                 let flushed_store =
                     self.create_and_insert_store(slot, aligned_total_size, "flush_slot_cache");
+                let hashes_refs = hashes.iter().collect::<Vec<_>>();
```

Comment:
here again we make a vec of refs. I'm looking into whether I can avoid this.

Reply:
the hash is held inside the slot cache cached items. rwlock<option<hash>>. so, hard to share a ref to it.